### PR TITLE
Fix false error caused by xunit.console.uwp output and other log issues

### PR DESCRIPTION
**Pull request by crummel**
MSBuild looks for "ERROR" in the output to mark a build step failed.  This removes the string from our output so we don't trigger its error detection.
Additionally, this fixes the output from the XUnit UWP app to correctly include the arguments passed and fixes the return code of xunit.console.uwp to be an error if the UWP app failed to launch or timed out.

### CODE REVIEW

**Comment by crummel** [comment]
@weshaggard @MattGal

File: src/xunit.console.uwp/HostedAppxTest.cs [hunk 1]
```diff
@@ -139,24 +139,29 @@ public int Run(bool debug, TimeSpan timeout)
             Console.WriteLine("UWP Activation HRESULT: " + hr);
             if (hr == 0)
             {
-            var p = Process.GetProcessById(pid.ToInt32());
-            Console.WriteLine("Running {0} in process {1} at {2}", p.ProcessName, pid, DateTimeOffset.Now);
-            while (timer.ElapsedMilliseconds < timeout.TotalMilliseconds && !p.HasExited)
-            {
-                Thread.Sleep(1000);
+                var p = Process.GetProcessById(pid.ToInt32());
+                Console.WriteLine("Running {0} in process {1} at {2}", p.ProcessName, pid, DateTimeOffset.Now);
```
Comment (MattGal) [inline src/xunit.console.uwp/HostedAppxTest.cs]: Interpolated strings are cool. 
like: 
'''Console.WriteLine($"Running {p.ProcessName} in process {pid} at {DateTimeOffset.Now}" );

**Comment by MattGal** [comment]
LGTM.   I'm curious why the ERROR causes me an error and not your local setup, did you figure that out?

File: src/xunit.console.uwp/HostedAppxTest.cs (hunk 1 above)
Comment (crummel) [inline src/xunit.console.uwp/HostedAppxTest.cs]: Indeed they are.  Updated.

**Comment by crummel** [comment]
No idea, the errors showed up on the clean machine the first time I tried.  I'm probably just going to write off the box.

**Pull request merged**